test/view_component/integration_test.rb

```diff
@@ -293,4 +293,21 @@ class IntegrationTest < ActionDispatch::IntegrationTest
       assert_select "title", "Component Previews for preview_component"
     end
   end
+
+  if Rails.version.to_f < 6.1
```

Comment:
Can we add test cases for `render_component_to_string` and `render_component` from a controller?

Reply:
Done in c3cb1be1f48dfcaca5a5509396326934114bd2d1